tests/networks/test_build.py

```diff
@@ -56,3 +58,59 @@ def test_directed_network():
     assert (
         directed_network.number_of_edges() == 2 * undirected_network.number_of_edges()
     )
+
+
+def test_jaccard_similarity():
+    """tests that when the jaccard similarity is used as an edge attribute it returns the correct value"""
+    sequence = [
+        ["I", "went", "to", "the", "party"],
+        ["i", "had", "fun", "at", "the", "party"],
+    ]
+    network = build_coocc(sequence, edge_attributes=["jaccard_similarity"])
+    attrs = nx.get_edge_attributes(network, "jaccard_similarity")
+    if ("went", "party") in attrs.keys():
+        assert attrs[("went", "party")] == 1 / 2
+    else:
+        assert attrs[("party", "went")] == 1 / 2
```

Comment:
I'm not sure if it matters too much for tests, but if you were going to interrogate the results of the tests, it might be slightly easier (albeit more boring!) to use ["a", "b", "c", ...] etc.